pkg/skaffold/util/http_test.go

```diff
@@ -34,8 +34,8 @@ func TestDownload_UserAgent(t *testing.T) {
 	defer ts.Close()
 
 	// although we don't have a version number in tests, the user-agent string still
-	// has `skaffold/<GOOS>/<GOARCH>/`
-	testutil.CheckDeepEqual(t, true, strings.HasPrefix(version.UserAgent(), "skaffold/"))
+	// has `skaffold (<GOOS>)`
```

Comment:
This does have GOARCH: you're including `platform` below:

	if version == "" {
		// likely running under tests
		return fmt.Sprintf("skaffold (%s)", platform)
	}
```suggestion
	// has `skaffold (<GOOS>/<GOARCH>)`
```